src/notifications.c

```diff
@@ -296,6 +296,16 @@ void ShardingEvent(RedisModuleCtx *ctx, RedisModuleEvent eid, uint64_t subevent,
   }
 }
 
+
+void ShutdownEvent(RedisModuleCtx *ctx, RedisModuleEvent eid, uint64_t subevent, void *data) {
+  if (eid.id != REDISMODULE_EVENT_SHUTDOWN) {
+    RedisModule_Log(RSDummyContext, "warning", "Bad event given, ignored.");

```

Comment:
Seems we can just silently skip non-interesting events and avoid logging them as "Bad"